### ISSUE TITLE
[Feature]: API Client can see that a CF Build + kpack Image & kpack Build results in a CF Droplet

### ISSUE THREAD

**Issue by gnovv** [cloudfoundry/cf-k8s-controllers#43]
### Blockers/Dependencies

BLOCKED BY: https://github.com/cloudfoundry/cf-k8s-controllers/issues/42
BLOCKED BY: https://github.com/cloudfoundry/cf-k8s-controllers/issues/52

### Background

As an API Client
I want a CF Build along with its kpack Image & Build CRs to result in a CF Droplet automatically
So that I can look up the latest built image by inspecting the CF Build and its Droplet

### Acceptance Criteria

GIVEN that I am targeting a k8s cluster
AND that I have followed the README instructions to install the Controllers and set up the kpack Registry Credentials and staging CRs
WHEN I apply the example CFApp, CFPackage, and CFBuild resources in config/samples directory
THEN I should see all those operations succeed
WHEN I fetch the yaml for the CF Build CR after its kpack Build completes successfully
THEN I should see that it now contains the Droplet's image

### Dev Notes
- Droplet processTypes and ports will be updated at the same time in a future story https://github.com/cloudfoundry/cf-k8s-controllers/issues/44
- Feel free to refer to the spike implementations of the [Build Controller](https://github.com/cloudfoundry/cf-crd-explorations/blob/main/controllers/build_controller.go) and the [cf-to-kpack Build Controller](https://github.com/cloudfoundry/cf-crd-explorations/blob/main/controllers/cf_kpack_build_controller.go) for inspiration
- Updating the Droplet's Process fields will be handled in a followup issue
 - For now ignore when a kpack Image is rebuilt and a new kpack Build is created after the first one, do not overwrite the droplet image

**Comment by gnovv** [comment]
- [ ] Happy Path
  - [ ] Unit Test
  - [ ] Implementation
- [ ] Unhappy Path
  - [ ] Unit Test
  - [ ] Implementation
- [ ] Decide if Integration Tests are required
- [ ] Test in real cluster
- [ ] Create PR
- [ ] Merge PR
